WebRTCPlugin/stdafx.h

```diff
@@ -25,3 +25,5 @@
 #include <atlctl.h>
 
 #include <string>
+
+#pragma warning (disable : 4267 4244)
```

Comment:
fix the warnings instead of hiding

Reply:
Agreed. Only had to disable the warnings to declutter the logs that showed 350+ errors at the time, but then I had to hand over the code in a hurry before cleaning up.
Nonetheless, the warnings are essentially shown during x64 build, which gives the basic "use size_t instead of int .. bla bla". No biggie, but I hadn't been tasked to specifically look at it, yet.